config/crd/kustomization.yaml

```diff
@@ -17,9 +17,9 @@ patchesStrategicMerge:
 
 # [CERTMANAGER] To enable webhook, uncomment all the sections with [CERTMANAGER] prefix.
 # patches here are for enabling the CA injection for each CRD
-- patches/cainjection_in_clusteradmissionpolicies.yaml
-- patches/cainjection_in_policyservers.yaml
-- patches/cainjection_in_admissionpolicies.yaml
+# - patches/cainjection_in_clusteradmissionpolicies.yaml
+# - patches/cainjection_in_policyservers.yaml
+# - patches/cainjection_in_admissionpolicies.yaml
```

Comment:
I'm sorry because I know I commented this. I was wondering why our CRD's had this annotation, thinking it was some mistake on our side -- copy/paste etc.

However, I have created a new kubebuilder project and I can see the patch targeting the `CustomResourceDefinition`.

I still don't understand the need for that, have to check in order to give a review.

Reply:
Given that they are required for cert-manager, I think we should leave them uncommented (sorry!).

They come with the direct templating from kubebuilder, so I would not touch them. It's required for the webhooks (the kubewarden ones -- clusteradmissionpolicy, admissionpolicy, policy-server) to be served over TLS with cert-manager.